docs/content/en/docs/clustermgmt/security/certificate-management/script-renew-certs.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Manually Renew Cluster Certificates"
+linkTitle: "Script To Renew Certificates"
+description: >
+  Step-by-step guide to renew Kubernetes certificates on EKS Anywhere clusters using a script.
+weight: 20
+---
+
+Certificates for external etcd and control plane nodes expire after 1 year in EKS Anywhere. EKS Anywhere automatically rotates these certificates when new machines are rolled out in the cluster. New machines are rolled out during cluster lifecycle operations such as `upgrade`. If you upgrade your cluster at least once a year, as recommended, manual rotation of cluster certificates will not be necessary.
+
+This page shows the process for manually rotating certificates if you have not upgraded your cluster in 1 year.
+
+The following table lists the cluster certificate files:
+
+| etcd node             | control plane node       |
+|-----------------------|--------------------------|
+| apiserver-etcd-client | apiserver-etcd-client    |
+| ca                    | ca                       |
+| etcdctl-etcd-client   | front-proxy-ca           |
+| peer                  | sa                       |
+| server                | etcd/ca.crt              |
+|                       | apiserver-kubelet-client |
+|                       | apiserver                |
+|                       | front-proxy-client       |
+
+Commands below can be used for quickly checking your certificates expiring date:
+
+```bash
+# The expiry time of api-server certificate on you cp node
+echo | openssl s_client -connect ${CONTROL_PLANE_IP}:6443 2>/dev/null | openssl x509 -noout -dates
+
+# The expiry time of certificate used by your external etcd server, if you configured one
+echo | openssl s_client -connect ${EXTERNAL_ETCD_IP}:2379 2>/dev/null | openssl x509 -noout -dates
+```
+
+You can rotate certificates by following the steps given below. You cannot rotate the `ca` certificate because it is the root certificate. Note that the commands used for Bottlerocket nodes are different than those for Ubuntu and RHEL nodes.
+
+## Overview
+This script automates:
+
+- Certificate renewal for etcd and control plane nodes.
```

Comment:
Can we just remove the punctuation all throughout for bullet points? Not sure if we do that in other sections